core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -72,6 +77,28 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
           </li>
         </ul>
       </div>
+      <div ng-if="isWindowLarge==false" class="right-section">
+        <ul class="metrics">
+          <li>
```

Comment:
There's a lot of duplication here. Can this be reduced? If the challenge is in conditionally setting the values of  `layout` and `layout-align`, consider ng-attr:

    http://stackoverflow.com/questions/15696416/what-is-the-best-way-to-conditionally-apply-attributes-in-angular